api/proto/teleport/summarizer/v1/summarizer.proto

```diff
@@ -107,3 +109,35 @@ message InferencePolicySpec {
   // match the list of kinds will be summarized using this model.
   string filter = 3;
 }
+
+// SummaryState is the state of the summarization process.
+enum SummaryState {
+  SUMMARY_STATE_UNSPECIFIED = 0;
+  SUMMARY_STATE_PENDING = 1;
+  SUMMARY_STATE_SUCCESS = 2;
+  SUMMARY_STATE_ERROR = 3;
+}
+
+// Summary represents a summary of a session recording. This format is used to
+// store the summaries in the session storage and return it with gRPC.
+message Summary {
+  // sessionId is an ID of the session whose recording got summarized.
+  string session_id = 1;
+  // State is the state of the summarization process.
+  SummaryState state = 2;
+  // InferenceStartedAt is the time when the summarization process started.
+  google.protobuf.Timestamp inference_started_at = 3;
+  // InferenceFinishedAt is the time when the summarization process finished.
+  google.protobuf.Timestamp inference_finished_at = 4;
+  // Content is the main text content of the summary. Available if the state is
+  // SUMMARY_STATE_SUCCESS.
+  string content = 5;
```

Comment:
Is this content plain text or markdown? (Might be worth while calling out in the comment).